WoWsShipBuilder.Web/Features/Layout/TopMenu.razor

```diff
@@ -85,6 +85,11 @@
                 break;
         }
         <MudDivider DividerType="DividerType.Middle" FlexItem Vertical/>
+        <NavLink href="/settings" class="main-nav-link d-flex" ActiveClass="nav-link-active">
```

Comment:
Why is this after the link to the live/pts version? If we do not want the settings on the right, we should treat it like a regular page which means putting it right after the links to ballistic charts, acceleration charts and the ship comparison but before the links to the other shipbuilder versions. Then there is also no need to add an additional divider since it is just a normal page like the others